README.rst

```diff
@@ -1,140 +1,47 @@
-This will make the ``artman`` command available on your system.
-
-.. _`virtualenvwrapper`: https://virtualenvwrapper.readthedocs.io/en/latest/
-.. _`virtualenv`: https://pypi.python.org/pypi/virtualenv
-
-
-Prerequisites
--------------
-1. Install `googleapis`_
-2. Install `toolkit`_
-3. Install Java
-4. Some languages may have additional dependencies; refer to the ``Dockerfile``
-   in this repository for canonical installation requirements.
-
-.. _`googleapis`: https://github.com/googleapis/googleapis
+.. _`Protocol Buffers`: https://developers.google.com/protocol-buffers/
+.. _`service configuration`: https://cloud.google.com/service-management/overview#service_configurations
 .. _`toolkit`: https://github.com/googleapis/toolkit
 
 
-Usage
------
-
-Before you can use artman, you will need a configuration file. You can run
-``configure-artman`` to create a simple configuration file.
-
-For building a GAPIC (the most common task), the usage looks like:
-
-.. code::
-
-    artman --api pubsub --language python
-
-Artman also takes a ``--publish`` argument to decide where to stage the
-code. Using ``--publish github`` will create a pull request on GitHub
-automatically.
-
-
-Python Versions
----------------
-
-artman is currently tested with Python 2.7, Python 3.4, Python 3.5,
-and Python 3.6.
-
-.. note::
-
-    The authors of this README are humans, and this is an exceptionally
-    easy spot to end up out of date.
-
-    Our ``nox.py`` and ``.circleci/config.yml`` files are the real source
-    of truth for what versions of Python we test against.
-
-
-Contributing
-------------
-
-Contributions to this library are always welcome and highly encouraged.
-
-See the `CONTRIBUTING`_ documentation for more information on how to get
-started.
-
-.. _`CONTRIBUTING`: https://github.com/googleapis/artman/blob/master/CONTRIBUTING.rst
-
-
-Versioning
-----------
-
-This library follows `Semantic Versioning`_
-
-It is currently in major version zero (``0.y.z``), which means that anything
-may change at any time and the public API should not be considered
-stable.
-
-.. _`Semantic Versioning`: http://semver.org/
-
-
-Details
--------
+Installing
+==========
 
-For detailed documentation of the modules in artman, please watch
-`DOCUMENTATION`_.
+If your goal is just to use ``artman`` (rather than contribute to it), a
+standard pip install is probably not the right thing; we recommend
+the use of `pipsi`_ instead.
 
-.. _`DOCUMENTATION`: https://googleapis-artman.readthedocs.org/
+View our `installation guide`_ to get going.
 
+.. _`pipsi`: https://github.com/mitsuhiko/pipsi
+.. _`installation guide`: https://googleapis-artman.readthedocs.io/latest/installing.html
 
-License
--------
+Documentation
+=============
 
-BSD - See `LICENSE`_ for more information.
+Documentation is available on `Read the Docs`_.
 
-.. _`LICENSE`: https://github.com/googleapis/artman/blob/master/LICENSE
+.. _`Read the Docs`: https://googleapis-artman.readthedocs.io/
```

Comment:
The link is broken. I assume it will soon become available once the PR is merged and uploaded to readthedocs?

Reply:
Correct. I already set up RTD but it builds from master.

Reply:
The docs as built from this branch are [here](http://googleapis-artman.readthedocs.io/en/readme/).